Do not build tests by default to avoid having to depend on Qt5.
--- CMakeLists.txt.orig	2024-10-17 14:54:57 UTC
+++ CMakeLists.txt
@@ -21,8 +21,6 @@ set(REQUIRED_QT_VERSION 6.5.0)
 
 set( oxygen_icon_dirs  8x8 16x16 22x22 32x32 48x48 64x64 128x128 256x256 )
 
-add_subdirectory(autotests)
-
 ########### install files ###############
 
 set(OXYGEN_INSTALL_DIR ${KDE_INSTALL_ICONDIR}/oxygen/base)
